lang/zh_CN.lang

```diff
@@ -608,11 +757,11 @@ value.BLOOM_RADIUS.5=高
 value.BLOOM_RADIUS.6=非常高
 value.BLOOM_RADIUS.7=最大
 
-option.LENS_FLARE=镜头光晕
-option.LENS_FLARE.comment=启用镜头光晕。
+option.LENS_FLARE=光晕
```

Comment:
looking at google translate, old is "lens flare" while the new one is just "flare". please recheck.